tests/unit-tests/corrections.php

```diff
@@ -0,0 +1,587 @@
+<?php
+/**
+ * Test_Corrections class.
+ *
+ * @package Newspack
+ */
+
+use Newspack\Corrections;
+
+/**
+ * Class Test_Corrections
+ *
+ * @group corrections
+ */
+class Test_Corrections extends WP_UnitTestCase {
+
+	/**
+	 * Holds a post ID to work with.
+	 *
+	 * @var int
+	 */
+	protected static $post_id;
+
+	/**
+	 * Set up test fixtures.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		if ( ! defined( 'NEWSPACK_CORRECTIONS_ENABLED' ) ) {
+			define( 'NEWSPACK_CORRECTIONS_ENABLED', true );
+		}
+
+		Corrections::init();
+
+		self::$post_id = $this->factory->post->create( [ 'post_type' => 'post' ] );
```

Comment:
The direct use of the `factory` field is deprecated. Not a big deal, but also very easy to fix by adding `()`: There is one more in the file further down too.

```suggestion
		self::$post_id = $this->factory()->post->create( [ 'post_type' => 'post' ] );
```